webpack.config.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     libraryTarget: 'var'
   },
   externals: {
+    'text-encoding': 'TextEncoder',
```

Comment:
Almost forgot this bit. If I remember correctly, `text-encoder` was the node shim for the in-browser `TextEncoder` object. Added this line to reflect that.